common/element_handle.go

```diff
@@ -1176,7 +1177,8 @@ func (h *ElementHandle) setChecked(apiCtx context.Context, checked bool, p *Posi
 	return nil
 }
 
-func (h *ElementHandle) Screenshot(opts goja.Value) goja.ArrayBuffer {
+// Screenshot will instruct Chrome to save a screenshot of the current element and save it to specified file.
+func (h *ElementHandle) Screenshot(opts goja.Value, filePersister *storage.LocalFilePersister) goja.ArrayBuffer {
```

Comment:
Should we just say `fp` or `lfp` since the type makes it clear that it's persister? No strong opinion.

Reply:
Resolved in 6868b5d16deb867fd64483336a1b6541c617c3cf